### DISCUSSION TITLE
Possibilities of Environments for Simulating Other Academic Events

### DISCUSSION THREAD

**Discussion by chengzr01** [ulab-uiuc/research-town#64] [Ideas]
This idea came from a discussion with Tao Feng. Based on our current framework, can we implement a different environment for simulating other academic events after submitting and reviewing a paper, such as _promoting a work_, _sharing a paper_, or _organizing a workshop_ in different contexts? Such environments might help people simulate the societal impacts of these events after paper submissions.

**Comment by lwaekfjlk** [comment]
sounds interesting! definitely a TODO. I think it can be done by adding more env.py.
